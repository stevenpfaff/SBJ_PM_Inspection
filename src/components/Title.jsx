import logo from '../assets/sbj.png'

const Title = ({ workOrderNumber = '' }) => (
  <>
    <h1>
      <img id='logo' src={logo} />
      PM Inspection Checklist
    </h1>
    {!!workOrderNumber && <h2>Work Order #{workOrderNumber}</h2>}
  </>
)

export default Title
