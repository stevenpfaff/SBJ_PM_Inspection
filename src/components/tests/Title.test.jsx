import { render } from '@testing-library/react'

import Title from '../Title'

describe('Title', () => {
  it('should match snapshot when workOrderNumber is empty', () => {
    const { container } = render(<Title />)

    expect(container).toMatchSnapshot()
  })

  it('should match snapshot when workOrderNumber is defined', () => {
    const { container } = render(<Title workOrderNumber='123456' />)

    expect(container).toMatchSnapshot()
  })
})
