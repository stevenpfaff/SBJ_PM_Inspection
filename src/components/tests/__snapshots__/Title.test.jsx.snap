// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match snapshot when workOrderNumber is defined 1`] = `
<div>
  <h1>
    <img
      id="logo"
      src="sbj.png"
    />
    PM Inspection Checklist
  </h1>
  <h2>
    Work Order #
    123456
  </h2>
</div>
`;

exports[`Title should match snapshot when workOrderNumber is empty 1`] = `
<div>
  <h1>
    <img
      id="logo"
      src="sbj.png"
    />
    PM Inspection Checklist
  </h1>
</div>
`;
